src/main/java/uk/gov/cshr/civilservant/domain/Question.java

```diff
@@ -34,9 +38,14 @@
     @Column(length = 500)
     private String why;
 
-    @ManyToMany(cascade = CascadeType.ALL)
-    private Set<Choice> choices = new HashSet<>();
+    @OneToMany (mappedBy = "question", fetch = FetchType.EAGER, cascade = CascadeType.ALL)
```

Comment:
Not saying it needs to be changed, but let's keep an eye on this Eager loading. In the past, It's caused us performance issues elsewhere in the app.